xml/System/Activator.xml

```diff
@@ -126,7 +126,7 @@
       </Parameters>
       <Docs>
         <param name="assemblyName">The name of a file that contains an assembly where the type named <paramref name="typeName" /> is sought.</param>
-        <param name="typeName">The name of the preferred type.</param>
+        <param name="typeName">The name of the type.</param>
```

Comment:
I have deleted "preferred" from these descriptions while I was on it. There is nothing "preferred" about these type names.